modules/python/python-dest.c

```diff
@@ -448,6 +448,10 @@ _py_construct_message(PythonDestDriver *self, LogMessage *msg, PyObject **msg_ob
   else
     {
       *msg_object = py_log_message_new(msg);
+
+      GlobalConfig *cfg = log_pipe_get_config(&self->super.super.super.super);
+      if (cfg_is_config_version_older(cfg, VERSION_VALUE_4_0))
```

Comment:
you should probably use cfg_is_typing_feature_enabled() instead of using the explicit version number.

Reply:
Thanks, fixed.